api/models/graphite.go

```diff
@@ -52,6 +52,42 @@ func (gr GraphiteRender) Validate(ctx *macaron.Context, errs binding.Errors) bin
 	return errs
 }
 
+type GraphiteTags struct {
+	Filter string `json:"filter"`
+	From   int64  `json:"from"`
+}
+
+type GraphiteTagsResp []GraphiteTagResp
+
+type GraphiteTagResp struct {
+	Tag string `json:"tag"`
+}
+
```

Comment:
the modeling here seems a bit weird. a named structure with 1 string member? 
can we do something like `type GraphiteTagsResp []string` or do it like you did `GraphiteTagFindSeriesResp` ?

Reply:
hmmm... then how could i end up with the same format as graphite outputs?
the output needs to be of this format:
```
[
  {
    "tag": "name"
  },
  {
    "tag": "datacenter"
  },
  {
    "tag": "rack"
  },
  {
    "tag": "server"
  }
]```

Reply:
oh well if that's a requirement, not much we can I do think.
Is there a reason the output format can't just be like this? do we expect to add more fields for each tag later?
```
[ "name", "datacenter", "rack", "server"]
```

Reply:
yes, it's designed to be extensible